File: DigiconValleyTest/app/src/main/java/com/shoaib/digiconvalleytest/MainActivity.java
package com.shoaib.digiconvalleytest;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.widget.FrameLayout;
import android.widget.TableLayout;

import com.google.android.material.tabs.TabLayout;
import com.shoaib.digiconvalleytest.fragment.OrdersFragment;
import com.shoaib.digiconvalleytest.fragment.PastDueFragment;
import com.shoaib.digiconvalleytest.fragment.UpcomingFragment;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }
}

File: AuthMVVM/app/src/main/java/com/shoaib/authmvvm/MainActivity.kt
package com.shoaib.authmvvm

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.shoaib.authmvvm.authentication.AuthActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        finish()
        startActivity(Intent(this, AuthActivity::class.java))

    }
}